docs/docsite/rst/dev_guide/developing_locally.rst

```diff
@@ -5,41 +5,45 @@
 Adding modules and plugins locally
 **********************************
 
-.. contents::
-   :local:
+The easiest, quickest, and the most popular way to extend Ansible is to use a module or a plugin. You can create them or copy existing ones for local use. You can store a local module or plugin on your Ansible control node and share it with your team or organization. You can also share a local plugin or module by embedding it in a role and publishing it on Ansible Galaxy. If you are using roles on Ansible Galaxy, then you are already using local modules and plugins without realizing it. 
 
-The easiest, quickest, and most popular way to extend Ansible is to copy or write a module or a plugin for local use. You can store local modules and plugins on your Ansible control node for use within your team or organization. You can also share a local plugin or module by embedding it in a role and publishing it on Ansible Galaxy. If you've been using roles off Galaxy, you may have been using local modules and plugins without even realizing it. If you're using a local module or plugin that already exists, this page is all you need.
+If you are using an existing module or plugin but Ansible can’t find it, this page is all you need. However, if you want to create a plugin or a module, go to :ref:`Developing plugins` and :ref:`Ansible module development: getting started` topics and then return to this page to know how to add it locally. 
```

Comment:
```suggestion
If you are using an existing module or plugin but Ansible can not find it, this page is all you need. However, if you want to create a plugin or a module, go to :ref:`Developing plugins` and :ref:`Ansible module development: getting started` topics and then return to this page to know how to add it locally. 
```

Reply:
Thank you for the review @Akasurde. I edited the content and committed the changes in PR https://github.com/ansible/ansible/pull/70659.